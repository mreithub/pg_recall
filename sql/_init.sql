--
-- creates some helper functions used by the other tests
--

CREATE FUNCTION pretendToWait(intv INTERVAL) RETURNS VOID AS $$
BEGIN
	UPDATE recall.config_log SET _log_time = tstzrange(LOWER(_log_time) - intv, UPPER(_log_time) - intv);
END;
$$ LANGUAGE plpgsql;


