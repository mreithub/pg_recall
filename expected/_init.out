--
-- creates some helper functions used by the other tests
--
CREATE FUNCTION pretendToWait(intv INTERVAL) RETURNS VOID AS $$
BEGIN
	UPDATE recall.config_log SET _log_start = _log_start - intv, _log_end = _log_end - intv;
END;
$$ LANGUAGE plpgsql;
